compiler/src/iree/compiler/Codegen/Dialect/LoweringConfig.td

```diff
@@ -113,18 +113,22 @@ def IREECodegen_TranslationInfoAttr :
 
   let assemblyFormat = [{
     `<` `` $passPipeline
-    (`pipeline_depth` `=` $softwarePipelineDepth^)? `>`
+    (`pipeline_depth` `=` $softwarePipelineDepth^)?
+    (`store_stage` `=` $softwarePipelineStoreStage^)? `>`
```

Comment:
That makes sense, I think in the future we will have a bunch of different schedules we will probably want to make it an enum with schedule names.